Source/CarthageKit/DependencyCrawler.swift

```diff
@@ -0,0 +1,226 @@
+import Foundation
+import Result
+import ReactiveSwift
+
+/**
+Signals for DependencyCrawler events
+*/
```

Comment:
This comment doesn't make sense. How are these _signals_?

Also, we should use the `///` comments that we use everywhere else.

Reply:
Updated doc, and now use /// everywhere